mlflow/utils/uri.py

```diff
@@ -22,7 +22,10 @@ def is_http_uri(uri):
 
 
 def is_databricks_uri(uri):
-    """Databricks URIs look like 'databricks' (default profile) or 'databricks://profile'"""
+    """
+    Databricks URIs look like 'databricks' (default profile) or 'databricks://profile'
+    or 'databricks://secret_scope:secret_key_prefix'.
```

Comment:
what will get in first, the fix from '/' -> ':', or this PR? :-)

Reply:
Well, I haven't started yet :)